docs/en/user-guide.md

```diff
@@ -0,0 +1,42 @@
+# Dumpling manual
+
+**Dumpling** is a tool and a Go library for creating SQL dump (CSV/SQL format) from a MySQL-compatible database.
+
+It is intended to replace `mysqldump` and `mydumper` when targeting TiDB, as a result, its basic usage is similar to Mydumper. 
+
+The following table lists main parameters of dumpling.
+
+
+| Parameter |     |
+| --------| --- |
+| -B or --database | Dump the specified database. |
+| -H or --host | Host to connect to. (default 127.0.0.1) |
+| -t or --threads | Number of threads for concurrent backup. |
+| -r or --rows | Split table into chunks of this many rows. (default unlimited) |
```

Comment:
This seems inconsistent with the Chinese version.
```suggestion
| -r or --rows | Number of rows the table is split into. (default: unlimited) |
```

Reply:
This description is from the CLI... I think this is more clear...

Reply:
OK